hrs-portlets-webapp/src/main/webapp/WEB-INF/tags/hrs/pagerNavBar.tag

```diff
@@ -7,20 +7,20 @@
 <%@ attribute name="hideNav" description="If the entire nav bar should be hidden" %>
 
 <div class="flc-pager-${position} dl-pager-navbar" style="display:none">
-  <ul class="dl-pager-bar fl-pager-ui">
+  <ul class="dl-pager-bar fl-pager-ui uw-pager-bar">
     <li class="flc-pager-previous dl-pager-previous">
-      <a href="javascript:;" title="Previous Page">&lt; Previous</a>
+      <a href="javascript:;" title="Previous Page" class="btn btn-default">Previous</a>
     </li>
     <li>
       <ul class="flc-pager-links dl-pager-links">
-        <li class="flc-pager-pageLink dl-pager-pageLink-default"><a href="javascript:;">1</a></li>
-        <li class="flc-pager-pageLink-skip dl-pager-pageLink-skip">&#8230;</li>
-        <li class="flc-pager-pageLink"><a href="javascript:;">2</a></li>
+        <li class="flc-pager-pageLink dl-pager-pageLink-default"><a href="javascript:;" class="btn btn-default">1</a></li>
+        <%-- <li class="flc-pager-pageLink-skip dl-pager-pageLink-skip">&#8230;</li> --%>
+        <li class="flc-pager-pageLink"><a href="javascript:;" class="btn btn-default">2</a></li>
       </ul>
     </li>
-    <li class="flc-pager-next dl-pager-next"><a href="javascript:;" title="Next Page">Next &gt;</a></li>
-    <c:if test="${showSummary == true}">
+    <li class="flc-pager-next dl-pager-next"><a href="javascript:;" title="Next Page" class="btn btn-default">Next</a></li>
+    <%-- <c:if test="${showSummary == true}">
       <li class="flc-pager-summary hrs-pager-summary"></li>
```

Comment:
Actually I meant to do this, didn't feel like it added much to the page but didn't want to completely delete it.